api/controllers/case.js

```diff
@@ -380,7 +381,7 @@ async function getCaseEditHttp(req, res) {
 async function getCaseNewHttp(req, res) {
   const params = parseGetParams(req, "case");
   params.view = "edit";
-  const article = JSON.parse(CASE_STRUCTURE);
+  const article = CASE_STRUCTURE;
```

Comment:
we need to parse this as json because fs.readFileSync at https://github.com/participedia/api/pull/492/files#diff-07780e70b8cc879c37de679a75e0e6bbR33 returns a buffer and the view needs json.

Reply:
Oh frack. My bad merge. I read the change backwards.

Also, we can return as a string if we pass an encoding, I forgot that nasty bit of Node weirdness.